yarn-project/end-to-end/src/guides/dapp_testing.test.ts

```diff
@@ -2,11 +2,11 @@
 import { getDeployedTestAccountsWallets } from '@aztec/accounts/testing';
 import { type AccountWallet, Fr, type PXE, TxStatus, createPXEClient, waitForPXE } from '@aztec/aztec.js';
 import { CheatCodes } from '@aztec/aztec.js/testing';
-// docs:end:imports
-// docs:start:import_contract
-import { TestContract } from '@aztec/noir-contracts.js/Test';
 // docs:end:import_contract
 import { TokenContract } from '@aztec/noir-contracts.js/Token';
+// docs:end:imports
+// docs:start:import_contract
+import { TestContract } from '@aztec/noir-test-contracts.js/Test';
 
```

Comment:
Heads up the docs magic comments are now imbalanced, this may break the docs build